src/main/java/racingcar/domain/entity/RacingGameResult.java

```diff
@@ -0,0 +1,36 @@
+package racingcar.domain.entity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import racingcar.domain.entity.car.Car;
+import racingcar.domain.entity.car.Cars;
+
+public class RacingGameResult {
+    private final List<Cars> carsHistory = new ArrayList<>();
+
+    public void logCurrentStatus(Cars cars){
+        carsHistory.add(cars);
+    }
+
+    public List<Car> getWinners() {
+        List<Car> cars = carsHistory.get(carsHistory.size() -1).getValue();
+        int maxPosition = getMaxPosition(cars);
+        return cars.stream()
+            .filter(car -> car.getPosition() == maxPosition)
+            .toList();
```

Comment:
Cars 일급 컬렉션만 추출하고, Cars에 메시지를 보내서 처리할 수 있을 것 같아요.


```suggestion
        return cars.getWinners();
```